src/styles/theme/oc-sidebar.scss

```diff
@@ -0,0 +1,77 @@
+.oc-sidebar {
+  @extend .uk-width-medium;
+
+  display: flex;
+  flex-flow: column wrap;
+  padding: $gutter 0;
+  height: 100%;
+  max-height: 100%;
+  background-color: $primary-background;
+  overflow-y: auto;
+
+  &-logo {
+    padding: 0 $small-gutter;
+    margin-bottom: $large-margin;
+    text-align: center;
+
+    &-img {
+      @extend .uk-width-small;
+    }
+
+    &-text {
+      @extend .uk-text-lead; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5442879)